subsys/suit/memory_layout/Kconfig

```diff
@@ -6,7 +6,7 @@
 
 config SUIT_MEMORY_LAYOUT_EXTMEM_ADDRESS_RANGE_START
 	hex
-	default 0x60000000 if SOC_NRF54H20
+	default 0x60000000 if SOC_SERIES_NRF54HX
 	default 0x12000000 if SOC_NRF52840
 	default 0x0 if BOARD_NATIVE_POSIX
```

Comment:
This should come from dts

Reply:
Unfortunately the EXMIF is not yet defined in DTS. To be aligned in the future (possibly along with the falsh_companion sample migration process).